specification/ml/_types/Datafeed.ts

```diff
@@ -68,15 +50,21 @@ export class DatafeedConfig {
    * The interval at which scheduled queries are made while the datafeed runs in real time. The default value is either the bucket span for short bucket spans, or, for longer bucket spans, a sensible fraction of the bucket span. For example: `150s`. When `frequency` is shorter than the bucket span, interim results for the last (partial) bucket are written then eventually overwritten by the full bucket results. If the datafeed uses aggregations, this value must be divisible by the interval of the date histogram aggregation.
    */
   frequency?: Timestamp
-  indexes?: string[]
+  /**
```

Comment:
The `indices`/`indexes` problem here is that on write we accept both but on read we always return `indices` and it's always an array of `IndexName`, not `Indices`.